cloudflare/resource_cloudflare_load_balancer.go

```diff
@@ -136,6 +144,149 @@ func resourceCloudflareLoadBalancer() *schema.Resource {
 	}
 }
 
+var rulesElem = &schema.Resource{
+	Schema: map[string]*schema.Schema{
+		"name": {
+			Type:         schema.TypeString,
+			Required:     true,
+			ValidateFunc: validation.StringLenBetween(1, 200),
+		},
+
+		"priority": {
+			Type:     schema.TypeInt,
+			Optional: true,
+			Computed: true,
+			Default:  nil,
+		},
+
+		"disabled": {
+			Type:     schema.TypeBool,
+			Optional: true,
+			Default:  false,
+		},
+
+		"condition": {
+			Type:     schema.TypeString,
+			Optional: true,
+			Default:  "",
```

Comment:
we don't need to be explicit about empty strings here; it will also throw off `d.GetOkExists` checks if we use them as we're setting a value.

```suggestion
```